source/extensions/filters/http/peer_metadata/config.proto

```diff
@@ -75,4 +75,13 @@ message Config {
 
   // True to enable sharing with the upstream.
   bool shared_with_upstream = 5;
+
+  // Additional labels to be added to the peer metadata.
+  // By default, following labels are added:
+  // - service.istio.io/canonical-name
```

Comment:
nit: these are bad labels to add here since they are already part of the typed data. Maybe we should add a more realistic example?

Reply:
I meant to say that these labels should not be added to additional labels, will update the doc to make it more clearer.